sklearn/linear_model/_bayes.py

```diff
@@ -658,8 +673,41 @@ def fit(self, X, y):
         dtype = X.dtype
 
         n_samples, n_features = X.shape
+
+        # Store original y scale for proper predictions
+        self.y_mean_ = np.mean(y)
+        self.y_scale_ = np.std(y)
+        if self.y_scale_ == 0:
+            self.y_scale_ = 1.0
+
+        # Make a copy of original y
+        y_orig = y.copy()
+
+        # Apply standardization if requested
+        if self.standardize:
+            # Standardize X
+            self.scaler_ = StandardScaler()
+            X = self.scaler_.fit_transform(X)
```

Comment:
I tend to prefer having scaling done in a pipeline before this step rather than here. I'm not sure if we've changed our opinion on that though (cc @lorentzenchr @ogrisel )